lib/utils.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var chainlib = require('chainlib');
+var utils = chainlib.utils;
+
+/**
+ * Bitcore's API does not implement toJSON in the standard way.
+ * This causes issues when doing a JSON.stringify on an object
+ * which contains Bitcore objects. This custom implmentation
+ * of stringify accounts for Bitcore objects.
+ * @param  {Object} obj
+ * @return {String} json
+ */
+utils.stringify = function(obj) {
+  return JSON.stringify(utils.expandObject(obj));
+}
+
+utils.expandObject = function(obj) {
```

Comment:
Will this be needed with `toJSON()` implemented as expected in: https://github.com/bitpay/bitcore/pull/1312 ?